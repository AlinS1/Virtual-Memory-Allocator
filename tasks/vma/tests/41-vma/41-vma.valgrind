==1746== Memcheck, a memory error detector
==1746== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==1746== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==1746== Command: /mnt/c/Users/alins/OneDrive/Desktop/SDA_tema1/vma
==1746== Parent PID: 1745
==1746== 
==1746== 
==1746== HEAP SUMMARY:
==1746==     in use at exit: 0 bytes in 0 blocks
==1746==   total heap usage: 4,811 allocs, 4,811 frees, 179,832 bytes allocated
==1746== 
==1746== All heap blocks were freed -- no leaks are possible
==1746== 
==1746== For lists of detected and suppressed errors, rerun with: -s
==1746== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
