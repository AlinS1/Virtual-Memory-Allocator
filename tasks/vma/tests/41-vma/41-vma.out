Total memory: 0x51C87E bytes
Free memory: 0x51C87E bytes
Number of allocated blocks: 0
Number of allocated miniblocks: 0
Total memory: 0x51C87E bytes
Free memory: 0x51C87A bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 1

Block 1 begin
Zone: 0x6A3 - 0x6A7
Miniblock 1:		0x6A3		-		0x6A7		| RW-
Block 1 end
Total memory: 0x51C87E bytes
Free memory: 0x51C87A bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 1

Block 1 begin
Zone: 0x6A3 - 0x6A7
Miniblock 1:		0x6A3		-		0x6A7	